### PR TITLE
Repository Setup - Typescript Installed, Linting and Pipeline Setup 

### PR DESCRIPTION
**Pull request by david-andrew-k**

### CODE REVIEW

**Comment by leeservice** [comment]
Can see the checks have passed - the Husky is barking! 
image

File: app.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+app.use(express.urlencoded({ extended: true }))
+
+app.listen(3000, () => {
+  console.log('Server listening on port 3000.')
+})
+
+// Home end point
+app.get('/', (req: Request, res: Response) => {
+  res.render('home', {
```
Comment (ccurran01) [inline app.ts:34]: Is there a html file called home?

File: app.ts [hunk 2]
```diff
@@ -1,8 +1,12 @@
-import * as express from 'express'
+// import * as express from 'express'
```
Comment (ccurran01) [inline app.ts:1]: Could remove this line

File: package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "eslint": "^8.45.0",
+    "eslint-config-standard-with-typescript": "^36.1.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^16.0.1",
+    "eslint-plugin-promise": "^6.1.1",
+    "husky": "^8.0.3",
+    "typescript": "^5.1.6"
+  },
+  "dependencies": {
+    "@types/nunjucks": "^3.2.3",
```
Comment (ccurran01) [inline package.json:40]: Could move this type up to dev dependencies

**Comment by david-andrew-k** [comment]
@ccurran01 That is fixed now. Thanks, David.

**Review by ccurran01 — APPROVED** [review]

**Pull request merged**